docs/tutorial/tutorial.md

```diff
@@ -1029,7 +1029,7 @@ Add a method called `jumpTo` to the Game class:
   jumpTo(step) {
     this.setState({
       stepNumber: step,
-      xIsNext: (step % 2) ? false : true,
+      xIsNext: !(step % 2),
```

Comment:
Cool, changes made. Sorry this ended up being such a nit picky PR. Thanks for the feedback!